test/import/FSHImporter.Profile.test.ts

```diff
@@ -155,12 +152,8 @@ describe('FSHImporter', () => {
         `;
 
         importText(input, 'Dupe.fsh');
-        expect(mockWriter.mock.calls[mockWriter.mock.calls.length - 2][0].message).toMatch(
-          /File: Dupe\.fsh.*Line: 7\D/s
-        );
-        expect(mockWriter.mock.calls[mockWriter.mock.calls.length - 1][0].message).toMatch(
-          /File: Dupe\.fsh.*Line: 8\D/s
-        );
+        expect(loggerSpy.getMessageAtIndex(1, true)).toMatch(/File: Dupe\.fsh.*Line: 7\D/s);
+        expect(loggerSpy.getLastMessage()).toMatch(/File: Dupe\.fsh.*Line: 8\D/s);
```

Comment:
I was wondering how you'd handle this scenario. I like the reverse-index approach you've used here.

Reply:
I asked people around me what they thought until someone mentioned it and it felt like the right approach!